### PR TITLE
Cloud changelog - March 16th

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by marianunez — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -4,12 +4,58 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Removed right-click hijacking on code blocks in messages.
+ - The order of the Leave Channel and Archive Channel channel settings were changed to be the same as in mobile.
+ - Added the condition to remove unread styling for archived channels and to filter archived channels from local data.
+
+#### Administration
+ - The invoice is now sent attached when an Admin upgrades to Cloud annual subscription.
+ - Removed Boards limits from Cloud Starter subscription.
+ - Enabled ``EnableOAuthServiceProvider`` by default.
+ - Export files now contain the read and unread status for channels.
+ - While upgrading a High Availability installation with rolling upgrades, there might be a situation where after upgrading all nodes, the System Console will show up as just one plugin is there and in disabled state. This is expected, and upon another restart of the first node which was upgraded, it should not show any plugins. That is the correct state. This happens due to the way plugins are initialized by copying over from prepackaged plugins to the plugins directory, and is an harmless artifact of that.
```
Comment (agnivade) [inline source/install/cloud-changelog.md:39]: This should only go in the release when product mode is enabled.

**Review by agnivade — COMMENTED** [review]
Anyways, this release is cancelled. So we can roll these over.

**Review by marianunez — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]

**Review by johndavidlugtu — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -4,12 +4,84 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - All post components were removed in favor of a unified approach.
+ - App bindings are now refreshed when a App plugin enabled event gets triggered.
+ - Improvements were added to the sidebar channel and category menus.
+ - Removed right-click hijacking on code blocks in messages.
+ - The order of the Leave Channel and Archive Channel channel settings were changed to be the same as in mobile.
+ - Added the condition to remove unread styling for archived channels and to filter archived channels from local data.
+ - Changed the collapsed post fade out effect to be less buggy.
+ - Users now have the ability to see the history of edited messages and to restore an old message version with the current version.
+
+#### Administration
```
Comment (johndavidlugtu) [inline source/install/cloud-changelog.md:44]: @amyblais it looks like the mention of Cloud Pro annual was lost in the shuffle... we need to mention that Cloud Professional monthly will be no longer be offered to new customers. Cloud Pro annual will be available.

Existing Cloud Pro monthly customers will get to keep their current subscription, but can move to Cloud Pro annual if they want to

cc: @marianunez

**Review by johndavidlugtu — CHANGES_REQUESTED** [review]

File: source/install/cloud-changelog.md (hunk 2 above)
Comment (marianunez) [reply source/install/cloud-changelog.md:44]: https://github.com/mattermost/docs/blob/aa59340622b5e1adb4620e40f212daf2b7678042/source/install/cloud-changelog.md#release-2023-02-02

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -21,25 +28,69 @@ Latest Mattermost Cloud releases:
  - Cloud Professional monthly will no longer be offered to new customers starting February 2, 2023.
```
Comment (johndavidlugtu) [inline source/install/cloud-changelog.md:28]: *February 23

Comment (amyblais) [reply source/install/cloud-changelog.md]: I put March 2nd which is when the next release is scheduled.

Comment (johndavidlugtu) [reply source/install/cloud-changelog.md]: Good catch -- thanks @amyblais

**Review by johndavidlugtu — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 4]
```diff
@@ -4,42 +4,93 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Removed Boards limits from Cloud Starter subscription.
+ - Enabled ``EnableOAuthServiceProvider`` by default.
+ - Export files now contain the read and unread status for channels.
+ - While upgrading a High Availability installation with rolling upgrades, there might be a situation where after upgrading all nodes, the System Console will show up as just one plugin is there and in disabled state. This is expected, and upon another restart of the first node which was upgraded, it should not show any plugins. That is the correct state. This happens due to the way plugins are initialized by copying over from prepackaged plugins to the plugins directory, and is an harmless artifact of that.
+ - Added the ``SentAt`` column to ``NotifyAdmin``.
+ - Updated ``NotifyAdmin.RequiredFeature`` column type to ``varchar(255)``.
+ - Updated ``NotifyAdmin.RequiredPlan`` column type to ``varchar(100)``.
+ - [Message Priority & Acknowledgement](https://docs.mattermost.com/configure/site-configuration-settings.html#message-priority) is now enabled by default for all instances.
+ - Added the ability to delete a workspace from the **System Console**.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:61]: Do we want to document the path to take to reach this option?

File: source/install/cloud-changelog.md [hunk 5]
```diff
@@ -4,42 +4,93 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added a new section in the **System Console** for products. For now, it only contains Boards-specific settings.
+ - Go version was bumped to v1.19.
  - GraphQL APIs are now correctly counted when measuring performance telemetry.
  - Boards cards are no longer mentioned as being limited in the **System Console**, the limits usage modal, the downgrade modal, or the left-hand side menu.
  - Removed an unused ``ProductLimits.Integrations``.
 
+#### Performance
+ - Reduced the rate that unreads are resynced when the window is focused from 10 seconds to 2 minutes.
+ - The center channel is no longer shown as loading when switching teams.
+Added logging fixes: empty ``short_message`` for Gelf formatter is no longer allowed and ``params.Host`` is now used over ``params.IP`` for syslog config.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:74]:
```suggestion
- Added logging fixes: empty ``short_message`` for Gelf formatter is no longer allowed and ``params.Host`` is now used over ``params.IP`` for syslog config.
```

File: source/install/cloud-changelog.md [hunk 6]
```diff
@@ -4,42 +4,93 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+
+### API Changes
+ - Added an ``exclude_files_count`` parameter to exclude file counts from the channel stats API.
+ - Added a new API endpoint ``GET api/v4/posts/[POST_ID]/edit_history``. Add upgrade notes from here.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:78]: "Add upgrade notes from here." -- is this placeholder text?

File: source/install/cloud-changelog.md [hunk 7]
```diff
@@ -4,42 +4,93 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added a new API endpoint ``DELETE /api/v4/cloud/delete-workspace``.
+
+### Bug Fixes
+ - Fixed new teams to use the updated translation for default channels after a config change.
+ - Fixed a layout issue in the System Console for smaller-sized tablets.
+ - Fixed an issue where a "plugin configured with a nil SecureConfig" warning was logged when starting each plugin.
+ - Fixed an issue where portal availability was checked when not on enterprise edition.
+ - Fixed a 404 error from requests to ``/api/v4/system/notices/`` on page load.
+ - Fixed an issue where OpenId Connect was configurable for Cloud Starter licenses.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:87]:
```suggestion
 - Fixed an issue where OpenID Connect was configurable for Cloud Starter licenses.
```

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline.

**Review by stylianosrigas — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 8]
```diff
@@ -4,42 +4,93 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Updated Chromium minimum supported version to 110+.
+ - Updated Edge minimum supported version to 110+.
 
 ### Highlights
 
 #### Annual Cloud Subscriptions
  - On the purchase modal, admins are now able to buy an annual cloud subscription starting from their current user count.
  - The **System Console > Billing & Account > Subscriptions** page now reflects whether the plan is monthly or annual.
- - Cloud Professional monthly will no longer be offered to new customers starting February 2, 2023.
+ - Cloud Professional monthly will no longer be offered to new customers starting March 2, 2023.
```
Comment (justinegeffen) [inline source/install/cloud-changelog.md:28]:
```suggestion
 - Cloud Professional monthly will no longer be offered to new customers starting March 16, 2023.
```

**Comment by cwarnermm** [comment]
@amyblais - Is this PR one we can safely close at this point, or does it form the basis of the next Cloud changelog?

**Comment by amyblais** [comment]
@cwarnermm I will add content for the next release to this PR.

**Review by cwarnermm — APPROVED** [review]

**Review by johndavidlugtu — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]

**Pull request merged**